src/libraries/System.DirectoryServices/src/System/DirectoryServices/ActiveDirectory/TrustHelper.cs

```diff
@@ -961,9 +961,13 @@ internal static string CreateTrustPassword()
             byte[] buf = new byte[PASSWORD_LENGTH];
             char[] cBuf = new char[PASSWORD_LENGTH];
 
+#if NETCOREAPP || NETSTANDARD2_1
```

Comment:
This library doesn't build for either of these configurations. There's no benefit to adding this.